### PR TITLE
I really only seeded database I haven't built the database to make the api calls

### PR DESCRIPTION
**Pull request by dkancalade**
oops, I forgot to write about this.

conained are the db seed file and the db helper.  
that is where most of my work is done.  

It's kind of big.  

-  I realize I didn't write comments on the seed.js file so I will do that before I close this request

### CODE REVIEW

**Comment by mervinpan** [comment]
@dkancalade Can you provide a sample output of the products with the links? I think you may have some overlapping URLs. 

Also are you going to create another connection to your database when your service is up and running? I believe it would be cumbersome to have to initiate the seed every time you start your service.

**Comment by dkancalade** [comment]
@mervinpan  Can you provide a sample output of the products with the links? I think you may have some overlapping URLs.


I do have some overlapping links.  I have 31 pictures and 100 products.  The only things that shouldn't be overlapping are links for individual products.  

The data is huge and I would have to screenshot console.logs to show you.  I don't know if that is what you'd really want.   

> Also are you going to create another connection to your database when your service is up and running? I believe it would be cumbersome to have to initiate the seed every time you start your service.

Yes I am going to create another connection.  This is not the db instance I am using for my api request.   This is just the seed file, so that means it is only ever applied to the database once.  Once the data is there, it is not used anymore, unless for some reason I want to do an entire db reset.

In other words I actually haven't created my databases index.html yet  My branch name for this pull request ended up being inaccurate.

Let me know if you still want me to show or verify anything else.

**Review by mervinpan — APPROVED** [review]
Looks good then

**Comment by dkancalade** [comment]
Cool thanks!
